packages/fxa-event-broker/lib/jwts.ts

```diff
@@ -47,10 +55,11 @@ export class JWT {
    * @param event
    */
   public generateSET(event: securityEvent): Promise<string> {
-    const now = Math.floor(Date.now() / 1000);
     const claims = {
       aud: event.clientId,
       events: event.events,
+      iat: Date.now() / 1_000,
```

Comment:
`1_000` looks like a typo

Reply:
I wanted this number to be in seconds, Date.now() is milliseconds so I divided by 1000. The underscore is a Typescript numeric separator. I suppose we could discuss how many zero's warrants using them, I'm pretty used to them in Rust as well.

Reply:
I think it's probably better to just use `1000`, that's used a lot throughout the codebase